.github/workflows/release.yml

```diff
@@ -54,20 +49,20 @@ jobs:
 
       - name: Update latest release version in docs
         run: |
-          mvn -B -ntp -pl docs -am generate-resources -Denforcer.skip -Dspotless.check.skip=true
+          mvn -B -ntp -pl docs -am generate-resources -Denforcer.skip -Dformatter.skip -Dimpsort.skip
           if ! git diff --quiet docs/modules/ROOT/pages/includes/attributes.adoc; then
             git add docs/modules/ROOT/pages/includes/attributes.adoc
             git commit -m "Update the latest release version ${{steps.metadata.outputs.current-version}} in documentation"
           fi
 
       - name: Maven release ${{steps.metadata.outputs.current-version}}
         run: |
-          mvn -B release:prepare -Prelease -Dspotless.check.skip=true -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
-          mvn -B release:perform -Dspotless.check.skip=true -Darguments=-DperformRelease -DperformRelease -Prelease -Dspotless.check.skip=true
-          mvn -B spotless:apply
```

Comment:
I noticed that `release` was changing the style of the `pom.xml`, I think it's safe to leave this, or have you tested this aspect already?

Reply:
Weird, I don't think it happens in other projects. At least I've never noticed it happen :)

Reply:
I disabled the standard formatting and we are using spotless here, is it an issue?

Reply:
I'd recommend use the standard formatting plugin (the one inherited from quarkuverse-parent), since that's the same all other extensions also use